### ISSUE TITLE
Code error in select_estimate and print_estimate?

### ISSUE THREAD

**Issue by mjbsp** [donaldRwilliams/BGGM#16]
While tracking down an error in roll_your_own, I discovered what I think are errors in both `select_estimate` and `print_estimate`.

In both of these functions there is one place where `fit` is called instead of `object` or `x`. I think this is just a leftover from a prior version of BGGM.

I believe that select_estimate should instead be 
```
function (object, cred = 0.95, alternative = "two.sided") 
{
  if (isFALSE(object$analytic)) {
    pcors <- object$post_samp$pcors[, , 51:(object$iter + 50)]   #CHANGE IS IN THIS LINE
    if (alternative == "two.sided") {
      lb <- (1 - cred)/2
      ub <- 1 - lb
      adj <- ifelse(apply(pcors, 1:2, quantile, lb) < 0 & 
                      apply(pcors, 1:2, quantile, ub) > 0, 0, 1)
    }
    else if (alternative == "greater") {
      lb <- (1 - cred)
      adj <- ifelse(apply(pcors, 1:2, quantile, lb) > 0, 
                    1, 0)
    }
    else {
      ub <- cred
      adj <- ifelse(apply(pcors, 1:2, quantile, ub) < 0, 
                    1, 0)
    }
  }
  else {
    if (alternative == "two.sided") {
      lb <- (1 - cred)/2
      ub <- 1 - lb
      z_stat <- abs(object$analytic_fit$inv_map/sqrt(object$analytic_fit$inv_var))
      adj <- ifelse(z_stat > qnorm(ub), 1, 0)
    }
    else if (alternative == "greater") {
      ub <- 1 - cred
      z_stat <- (-object$analytic_fit$inv_map)/sqrt(object$analytic_fit$inv_var)
      adj <- ifelse(z_stat > qnorm(ub, lower.tail = FALSE), 
                    1, 0)
    }
    else if (alternative == "less") {
      ub <- 1 - cred
      z_stat <- (object$analytic_fit$inv_map)/sqrt(object$analytic_fit$inv_var)
      adj <- ifelse(z_stat > qnorm(ub, lower.tail = FALSE), 
                    1, 0)
    }
  }
  pcor_adj <- adj * object$pcor_mat
  returned_object <- list(pcor_adj = pcor_adj, adj = adj, alternative = alternative, 
                          cred = cred, object = object)
  class(returned_object) <- c("BGGM", "select.estimate", "estimate", 
                              "select")
  returned_object
}
```

I believe that print_estimate should instead be:

```
function (x, ...) 
{
    cat("BGGM: Bayesian Gaussian Graphical Models \n")
    cat("--- \n")
    cat("Type:", x$type, "\n")
    cat("Analytic:", x$analytic, "\n")
    cat("Formula:", paste(as.character(x$formula), collapse = " "),  #CHANGE IS IN THIS LINE
        "\n")
    cat("Posterior Samples:", x$iter, "\n")
    cat("Observations (n):\n")
    cat("Nodes (p):", x$p, "\n")
    cat("Relations:", 0.5 * (x$p * (x$p - 1)), "\n")
    cat("--- \n")
    cat("Call: \n")
    print(x$call)
    cat("--- \n")
    cat("Date:", date(), "\n")
}
```

**Comment by donaldRwilliams** [comment]
Thank you for this. I actually just noted this yesterday while cleaning up all the examples. Noted that the object had to be named fit to work, and changed it accordingly.

It is fixed, but you will want to wait before reinstalling. I will post here when the developmental version can be reinstalled.

**Comment by donaldRwilliams** [comment]
The developmental version can now be installed. Changing everything over to `c++` turned into quite the bit of work, but it now passes. see here

[https://travis-ci.org/github/donaldRwilliams/BGGM/builds/686987661](current build)

One caveat is that I removed all the examples and will be putting them back in one by one. Note that I checked the estimate and roll your own workflow and that works nicely, although, as I mentioned, the examples have been removed.
